vnext/ReactWindowsCore/packages.config

```diff
@@ -1,6 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="boost" version="1.68.0.0" targetFramework="native" />
-  <package id="ReactNative.Hermes.Windows" version="0.1.6" targetFramework="native" Condition="$(USE_HERMES)' == 'true'" />
-  <package id="ReactNative.V8Jsi.Windows" version="0.2.7" targetFramework="native" Condition="'$(USE_V8)' == 'true'" />
+  <!-- 
```

Comment:
><!--  [](start = 2, length = 5)

This change breaks the RN usage for Office.
I think a better fix would be to change the ReactWindowsCore project into a shared items project that would remove any need for this file, and then:
- Change the MS.RN project to not depend on them.
- Change the desktop version (react-native-win32) to use these packages (it must not be deployed in the NPM)

Reply:
@vmoroz I think you meant to add a comment on this thread here but it's showing empty :) please reactivate this if needed